integration/spec_helper.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+require_relative "supports/logstash_test_helpers"
+require "fileutils"
+
+# Configure the test environment
+source = File.expand_path(File.join(File.dirname(__FILE__), ".."))
+integration_path = File.join(source, "integration_run")
+
+puts "[integration_spec] configure environment"
+
+if Dir.exists?(integration_path)
```

Comment:
I wonder if we should add  warning here and simply break. I imagine running a test, failing and then willing to run them again with the same dir. makes sense?